src/Result/LightningPayment.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+use BTCPayServer\Util\PreciseNumber;
+
+class LightningPayment extends AbstractResult
+{
+    public function getId(): ?string
+    {
+        return $this->getData()['id'];
+    }
+
+    public function getStatus(): ?string
+    {
+        return $this->getData()['status'];
+    }
+
+    public function getBolt11(): ?string
+    {
+        return $this->getData()['BOLT11'];
+    }
+
+    public function getPaymentHash(): ?string
+    {
+        return $this->getData()['paymentHash'];
+    }
+
+    public function getPreimage(): ?string
+    {
+        return $this->getData()['preimage'];
+    }
+
+    public function getCreatedAt(): ?int
+    {
+        return $this->getData()['createdAt'];
+    }
+
+    public function getTotalAmount(): ?PreciseNumber
```

Comment:
can't be NULL according to docs and your screenshot of the buggy response

Reply:
You're right this is never null, i will update